src/rest-server/docs/swagger.yaml

```diff
@@ -934,7 +934,7 @@ paths:
           $ref: '#/components/responses/NoJobError'
         '501':
           $ref: '#/components/responses/UnknownError'
-  '/api/v2/jobs/{user}~{job}/job-attempts/{attempt}':
+  '/api/v2/jobs/{user}~{job}/job-attempts/{attemptIndex}':
```

Comment:
let's also rename `attempt` in parameters

Reply:
done